README.md

```diff
@@ -123,6 +123,27 @@ Listed below are the steps to add these dependencies in a project.
  ``` 
        testImplementation group: 'org.hamcrest', name: 'hamcrest', version: '2.2'
  ```
+
+## Java Version Requirement
+Sapient AI Test Coder supports test generation for Java Version 8 through 17. To update java version for:
+
+- Gradle
+ ```
+ java {
+    sourceCompatibility = JavaVersion.VERSION_17
+    targetCompatibility = JavaVersion.VERSION_17
```

Comment:
for Java Version 8 to 17
[15:33](https://sapient-hcz8405.slack.com/archives/D04J1UHPJMA/p1684922626678819)
Follow below steps update java version for project: (edited) 
[15:34](https://sapient-hcz8405.slack.com/archives/D04J1UHPJMA/p1684922675206859)
we should not speicify the version
[15:34](https://sapient-hcz8405.slack.com/archives/D04J1UHPJMA/p1684922682758059)
we should just keep a placeholder version
[15:34](https://sapient-hcz8405.slack.com/archives/D04J1UHPJMA/p1684922684859049)
- Gradle
 ```
 java {
    sourceCompatibility = JavaVersion.VERSION_17
    targetCompatibility = JavaVersion.VERSION_17
}
 ```
- Maven
 ```
<properties>
    <maven.compiler.source>1.8</maven.compiler.source>
    <maven.compiler.target>1.8</maven.compiler.target>
  </properties>
 ```

Reply:
done